### DISCUSSION TITLE
Build libcurl with libevent

### DISCUSSION THREAD

**Discussion by ghost** [curl/curl#8165] [Q&A] [answered]
Hello,

Tell me please, i recently found out that libcurl has a multi_socket interface that works in conjunction with libvent.
The site even has an example: https://curl.se/libcurl/c/hiperfifo.html

But I don’t understand, is it necessary to somehow build libcur together with libvent?

If so, it is not clear how. For example, I look at Makefile.vc and MakefileBuild.vc - for windows for msvc, there is an indication of the flags - Zlib, OpenSSL, С-ares, nghttp2, but no flags -libevent.

Anyone could suggest how to use libcurl with Libevent?
In the official documentation kurl - unfortunately nothing is said about this.

**Comment by dfandrich** [comment, marked as answer]
libcurl doesn't change for libevent. Only the way you use libcurl changes.

**Reply by ghost** [reply]
that is, I just need to build a separate libevent and connect it to a project using curl? Right ?

**Reply by bagder** [reply]
Correct

**Reply by ghost** [reply]
Thank you!
